salon_creator/lib/screens/lesson_edit_screen.dart

```diff
@@ -0,0 +1,664 @@
+import 'dart:io';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:salon_creator/common/color.dart';
+import 'package:salon_creator/firebase/database.dart';
+import 'package:salon_creator/models/lesson.dart';
+import 'package:salon_creator/models/resource.dart';
+import 'package:salon_creator/widgets/custom_button.dart';
+import 'package:salon_creator/widgets/custom_dialog.dart';
+import 'package:salon_creator/widgets/custom_label.dart';
+import 'package:salon_creator/widgets/custom_textfield.dart';
+import 'package:salon_creator/widgets/lesson_card.dart';
+import 'package:salon_creator/widgets/custom_progress_indicator.dart';
+
+class LessonEditScreen extends StatefulWidget {
+  const LessonEditScreen({
+    this.lessonInfo,
+    Key key,
+  }) : super(key: key);
+  final LessonInfo lessonInfo;
+
+  @override
+  _LessonEditScreenState createState() => _LessonEditScreenState();
+}
+
+class _LessonEditScreenState extends State<LessonEditScreen> {
+  final dbHandler = DbHandler();
+  String path = "";
+  bool _isPublished = false;
+  bool _isFormFilled = false;
+  bool _operationInProgress = false;
+  bool _switchToNewThumbnail = false;
+  bool _switchToNewMedia = false;
+  String _storageThumbnailUrl;
+  String _storageMediaUrl;
+  List<Resource> _oldResourcesList = [];
+  List<Resource> _newResourcesList = [];
+  List<Resource> _currentResource = [];
```

Comment:
```suggestion
  List<Resource> _currentResourceList = [];
```
Using plural since it is a list of items